spec/spec_helper.rb

```diff
@@ -24,12 +22,24 @@ class SecondUser < Minidoc
   attribute :age, Integer
 end
 
+class ValidationsUser < User
```

Comment:
Heh yea, I realized belatedly that this was the perfect opportunity to try the feature where you change the target branch after opening the PR 😞